tools/runners/parallels.sh

```diff
@@ -190,6 +190,7 @@ function mktempdir {
 
   cp $_sdl_prepared/bin/hmi_capabilities.json $tmpdirname/bin/
   cp $_sdl_prepared/bin/log4cxx.properties $tmpdirname/bin/
+  cp $_sdl_prepared/bin/boostlogconfig.ini $tmpdirname/bin/
```

Comment:
```suggestion
  cp $_sdl_prepared/bin/boostlogconfig.ini $tmpdirname/bin/ 2>/dev/null
```

Using ATF against a Core build without boostlog prints this error a lot, can we suppress?

```
cp: cannot stat '/ram/prepared_env/sdl/bin/boostlogconfig.ini': No such file or directory
```

Reply:
Added in https://github.com/smartdevicelink/sdl_atf/pull/216/commits/314ae47670017992a0038790320432e3649187d5